src/utility/spi_drv.cpp

```diff
@@ -215,16 +215,16 @@ void SpiDrv::waitForSlaveSign()
 	while (!waitSlaveSign());
 }
 
-void SpiDrv::waitForSlaveReady(bool const feed_watchdog)
+void SpiDrv::waitForSlaveReady()
 {
-    unsigned long const start = millis();
+    unsigned long trigger_time = millis();
 	while (!waitSlaveReady())
     {
-        if (feed_watchdog) {
-            if ((millis() - start) < 10000) {
-                WiFi.feedWatchdog();
-            }
+        if (static_cast<int32_t>(trigger_time - millis()) <=0) {
```

Comment:
Okay, I see your point. However, can't it be written in a simplified way without needing to cast to `int32_t`? Something along the lines of
```C++
unsigned long start = millis();
/* ... */
if(feed_watchdog)
{
  unsigned long const now = millis();  
  if(now - start > 10000)
  {
    WiFi.feedWatchdog();
    start = now;
  }
}
```
?
Also `feed_watchdog` is there for purpose, only the BearSSL offloaded SSL calls take longe enough to warrant feeding the watchdog outside of `update()`.